packages/backend/src/tokens/interstitial.ts

```diff
@@ -26,6 +35,7 @@ export function loadInterstitialFromLocal(options: Omit<LoadInterstitialOptions,
         <script>
             window.__clerk_frontend_api = '${frontendApi}';
             window.__clerk_debug = ${JSON.stringify(debugData || {})};
+            ${domain ? `window.__clerk_domain = '${domain}'` : ''}
```

Comment:
❓ why don't we use `window.__clerk_domain = '${domain}'` ?

Reply:
@dimkl I think there is no need to include it if it doesn't exist, right?